prow/tide/tide.go

```diff
@@ -1532,7 +1532,20 @@ func prMeta(prs ...PullRequest) []prowapi.Pull {
 	return res
 }
 
-func sortPools(pools []Pool) {
+// TODO: should we memoize this?
+func getPRPriority(pr PullRequest, priorities []config.TidePriority) int {
+	for i, p := range priorities {
+		labels := sets.NewString(p.Labels...)
+		for _, l := range pr.Labels.Nodes {
+			if labels.Has(string(l.Name)) {
```

Comment:
Wouldn't it be more logical to require all labels in the priority setting?

Reply:
yes, the priority should only match if all of the labels in the priority are on the PR